modin/pandas/base.py

```diff
@@ -1473,17 +1473,29 @@ def kurt(self, axis=None, skipna=None, level=None, numeric_only=None, **kwargs):
         Returns:
             kurtosis : Series or DataFrame (if level specified)
         """
+        axis = self._get_axis_number(axis)
         if level is not None:
-            return self._default_to_pandas(
-                "kurt",
-                axis=axis,
-                skipna=skipna,
-                level=level,
-                numeric_only=numeric_only,
-                **kwargs,
+            from .series import Series
+
+            func_kwargs = {
+                "axis": 0,
+                "skipna": skipna,
+                "level": level,
+                "numeric_only": numeric_only,
+            }
+            if isinstance(self, Series):
+                return (
+                    self.groupby(level=level, axis=axis, sort=False)
+                    .apply(func="kurt", **func_kwargs)
+                    .rename(None)
+                )
+
+            return self.__constructor__(
+                query_compiler=self._query_compiler.apply(
+                    func=lambda df: df.kurt(**func_kwargs), axis=0
+                )
             )
```

Comment:
Will `return self.apply("kurt", **func_kwargs)` gives the same result? If yes, then it will help to avoid passing UDFs to the `query_compiler`

Reply:
Yes, it can solve this problem, but now it's blocked by https://github.com/modin-project/modin/pull/1657 PR.